src/uncontrast/pages/UploadNewPaper/ExperimentsSection/Paradigms.jsx

```diff
@@ -65,31 +59,14 @@ export default function Paradigms({
     }
   }, []);
   async function uniqSubmit(index) {
-    let chosenParadigm = fieldValues[index].sub_type
-      ? optionalParadigms.find(
-          (paradigm) => paradigm.sub_type === fieldValues[index].sub_type
-        ).id
-      : optionalParadigms.find(
-          (paradigm) => paradigm.name === fieldValues[index].specific
-        ).id;
-    handleSubmit(chosenParadigm, index);
+    let chosenParadigm = fieldValues[index].specific;
+    createUncontrastExperiments({ study_pk, chosenParadigm });
   }
 
-  const paradigmsWithSubtype = [
-    ...new Set(
-      optionalParadigms
-        .filter((paradigm) => paradigm.sub_type !== null)
-        .map((item) => item.name)
-    ),
-  ];
-
   const submitCondition = (index) => {
-    return [
-      fieldValues[index]?.main,
-      fieldValues[index]?.specific,
-      !paradigmsWithSubtype.includes(fieldValues[index]?.specific) ||
-        fieldValues[index]?.sub_type,
-    ].every((condition) => Boolean(condition) === true);
+    return [fieldValues[index]?.main, fieldValues[index]?.specific].every(
+      (condition) => Boolean(condition) === true
```

Comment:
Usually I use `!!` but I think Boolean is nicer...

Reply:
nice.. i didnt know !!